recipe/meta.yaml

```
@@ -27,7 +28,7 @@ requirements:
     - setuptools
     - numpy
   run:
-    - python
+    - python >=3.6,<4.0
```

Please remove the upper bound, we try to keep this open.
```suggestion
    - python >=3.6
```